package.json

```diff
@@ -4,48 +4,14 @@
   "description": "Provides a common starting point for all Element blocks",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
-    "build": "npm test && rollup -c rollup.config.prod.js",
-    "build:no-test": "rollup -c rollup.config.prod.js",
-    "start": "rollup -c rollup.config.dev.js --watch"
+    "test": "block-scripts test",
```

Comment:
Does this auto-run watch?

Reply:
No. CLI arguments are passed through to the block scripts, though, so `block-scripts test --watch` or `npm test -- --watch` will still work.

Reply:
I thought we were going to mimic the create-react-app module when it comes to the tests?

Reply:
They don't provide an easy way to run tests without watch mode, as far as I know, which I don't like. What about adding an additional `test:watch` script here?

Reply:
I'd rather pass ` --watch` than add `:watch` since I don't have to correctly time the shift key. When I'm typing quickly after opening a new terminal tab and type in the `npm run test:watch` command while AVN is loading the correct version of npm, it's an easy mistake to make. 

However, I would ague that there's not a lot of reasons to run the tests outside of watch mode.

Reply:
[Fixed.](https://github.com/volusion/element-BlockStarter/pull/8/commits/d17c823b3db3a421f8225d7e6b7d9b4214ae90db)

- `npm test` now runs `block-scripts test --watch`
- `npm run build` now runs `block-scripts test` without the watch option before running `block-scripts build`